Fixed #12939: Hard coded table names for MSSQL in RBAC migration

arogachev opened the pull request:
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #12939

Not sure if `dbo.` prefix is needed here, left it as it was, just removed hard coded values.

arogachev committed:
Fixed #12939: Hard coded table names for MSSQL in RBAC migration

samdark reviewed (changes requested):
Looks good. Need a line for CHANGELOG.

arogachev committed:
Updated CHANGELOG with #12939, removed duplicates

arogachev commented:
Sorry, forgot it. Added. 

Also some duplicates and extra newline appeared after https://github.com/yiisoft/yii2/commit/2d96da4e2933c1c8566731f1f9e5531bf4d71368. Removed them. I checked with global search of full lines in IDE in whole document (just in case) - indeed, these are complete duplicates.

arogachev commented:
Also one duplicate line contained extra dash causing break of list items formatting:

```
-- Enh #12748: Added Migration tool automatic generation reference column for foreignKey (MKiselev)
```

Fixed.

samdark commented:
Merged https://github.com/yiisoft/yii2/commit/fb8fd66a08294bdb2e5fc41159a938f701b25824. Thank you!
